app/services/nebraska/eligible_days_calculator.rb

```diff
@@ -23,7 +23,9 @@ def date_range
     def closed_days_by_month_until_date
       closed_days = 0
       date_range.each do |day|
-        closed_days += 1 unless child.business.eligible_by_date?(day)
+        child_business = @child.child_businesses.find_by(currently_active: true)
+        active_business = businesses.find(child_business.business_id)
```

Comment:
Why these two lines are needed(26 and 27)?

Wouldn't this be enough?
active_business = @child.child_businesses.find_by(currently_active: true)